unattended_installer/install_functions/installCommon.sh

```diff
@@ -178,6 +178,12 @@ function installCommon_changePasswordApi() {
         fi
     fi
 
+    for i in "${!api_users[@]}"; do
+        if [ "${api_users[i]}" == "wazuh" ] || [ "${api_users[i]}" == "wazuh-wui" ]; then
+            common_logger "The password for user ${api_users[i]} is ${api_passwords[i]}"
```

Comment:
I think we should change the message to print something like:
`The password for the ${api_users[i]} user is ${api_passwords[i]}`

Reply:
Done!